packages/tinacms/src/toolkit/fields/plugins/mdx-field-plugin/plate/index.tsx

```diff
@@ -57,7 +57,6 @@ export const RichEditor = (props: RichTextType) => {
   )
 
   // This should be a plugin customization
```

Comment:
**question**: Is this comment referring to the removed line immediately below it, or the 'paragraph' of code?

Reply:
I have reverted this change as discussed as it is not relevant to this PR